Lib/test/test_zlib.py

```diff
@@ -512,18 +512,7 @@ def test_odd_flush(self):
 
         # Try 17K of data
         # generate random data stream
-        try:
-            # In 2.3 and later, WichmannHill is the RNG of the bug report
-            gen = random.WichmannHill()
-        except AttributeError:
-            try:
-                # 2.2 called it Random
-                gen = random.Random()
-            except AttributeError:
-                # others might simply have a single RNG
-                gen = random
-        gen.seed(1)
-        data = gen.randbytes(17 * 1024)
+        data = random.randbytes(17 * 1024)
```

Comment:
It will take about 500 lines for hex-encoded or base64-encoded data (more if use the bytes literal). Generating 17KB of random data takes less than 0.0001 sec.

Reply:
@vstinner I've recently checked that it is not possible to install anything below python3.7 on m2 https://github.com/python/cpython/pull/110013#issuecomment-1739485985